spring-cloud-stream-binder-kafka-streams/pom.xml

```diff
@@ -62,5 +62,10 @@
 			<artifactId>spring-cloud-stream-binder-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-autoconfigure-processor</artifactId>
```

Comment:
@sobychacko ,

don't we need to revise our all other binder implementation moving them to the normal Spring Boot auto-configuration model?

I somehow believe that the current `spring.binders` is fully similar what `spring.factories` provides. Therefore such an optimization should be done in all the binders.

BTW, shound't this fix be back-ported?

Reply:
@artembilan, @olegz has already started on a change to use `spring.factories` instead of `spring.binders`, but preserved the usage of `spring.binders` for backward compatibility.

Yes, we need to backport this fix.  Thanks!